### PR TITLE
Run flow check on Windows

### PR DESCRIPTION
**Pull request by thangngoc89**
Flow 0.30.0 supported it. Yay

### CODE REVIEW

**Comment by thangngoc89** [comment]
need to investigate why Appveyor is failing with no useful information. It's fine on my laptop

**Comment by MoOx** [comment]
Everything is failing, see https://github.com/MoOx/phenomic/pull/549#issuecomment-230976904
This will require some work to update our code.

**Comment by thangngoc89** [comment]
@MoOx yeah I know about that and I'm fixing it. What I'm saying is flow can't run on Appveyor. I'm considering https://github.com/flowtype/flow-typed.

**Comment by thangngoc89** [comment]
Turn out flow-bin doesn't download flow.exe. Here is the log 

```
PS C:\projects\phenomic> npm rebuild flow-bin

> flow-bin@0.30.0 postinstall C:\projects\phenomic\node_modules\flow-bin
> node lib/install.js

npm :   Ã— No binary found matching your system. It's probably not supported.
At line:1 char:1
+ npm rebuild flow-bin
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (  Ã— No binary ... not supported.:S 
   tring) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
  Ã— flow binary test failed
flow-bin@0.30.0 C:\projects\phenomic\node_modules\flow-bin
```

**Comment by thangngoc89** [comment]
@MoOx good to merge :D

File: .flowconfig [hunk 1]
```diff
@@ -7,8 +7,8 @@
 .*/.nyc_output/.*
 
 [libs]
-./node_modules/flow-interfaces/interfaces/
 ./interfaces/
+./flow-typed/
```
Comment (MoOx) [inline .flowconfig]: Can't we just move flow-typed defs in "interfaces"? I don't want a mess at the root of the project :)

**Comment by MoOx** [comment]
@thangngoc89 not ok on appveryor :/
Super cool to handle that anyway!

**Comment by MoOx** [comment]
Oh yeah it's the autoshit error. Nevermind!

**Comment by MoOx** [comment]
I made the adjustment ;)

**Comment by thangngoc89** [comment]
Thanks a lot

**Pull request merged**